lang/semgrep-grammars/src/semgrep-kotlin/test/corpus/semgrep.txt

```diff
@@ -16,10 +16,11 @@ class $CLASS {
     (class_body
       (function_declaration
         (simple_identifier)
-        (parameter
-          (simple_identifier)
-          (user_type
-            (type_identifier)))
+        (function_value_parameters
```

Comment:
why this extra node?

Reply:
it was added via https://github.com/returntocorp/tree-sitter-kotlin/commit/f4eb099b34d1bf13b0e1dbc3e250199f605cdeb1, but not updated within our `semgrep-kotlin` test suite because we hadn't updated to that version of `tree-sitter-kotlin` yet